release/nightly_tests/dataset/autoscaling_gpu_compute.yaml

```diff
@@ -3,7 +3,8 @@
 cloud_id: {{env["ANYSCALE_CLOUD_ID"]}}
 region: us-west-2
 
-max_workers: 10
+advanced_configurations_json:
+    IamInstanceProfile: {"Name": "ray-autoscaler-v1"}
```

Comment:
What does this do? Do we need it for all of our compute YAMLs?

Reply:
This chaos is killing the ec2 instance instead of killing the raylet so I need this for proper IAM permissions.

Reply:
Should we add this to all our configs?

Reply:
Only add to those that do chaos tests

Reply:
Is it an issue if it's in a non-chaos test?  We use `autoscaling_gpu_compute` for the regular batch inference tests as well

Reply:
No. It's just IAM.